Archive/Arithcc.lean

```diff
@@ -321,7 +321,8 @@ theorem compiler_correctness :
   -- 5.III
   case sum =>
     rename_i e_s₁ e_s₂ e_ih_s₁ e_ih_s₂
-    simp
+    simp only [compile._eq_3, List.append_assoc, List.singleton_append, List.cons_append,
```

Comment:
I don't think we should be referring to `._eq_3` manually.

Reply:
ok I have reverted all the things that look like this.